### ISSUE TITLE
Error in sample.int

### ISSUE THREAD

**Issue by PyhaTanja** [ybrandvain/polyembryony#5]
> z <-runSim(n.gen = 100, fitness.effects = 1, dom.effects = 0 ,  gen.after.loss = 15,  gen.after.fix = 15 , polyemb.p0 = 0, introduce.polyem = 0)
[1] 1
[1] 2
Error in sample.int(n, size, replace = replace, prob = weight) : 
  NA in probability vector

**Comment by ybrandvain** [comment]
i fixed this by adding epsilon to fitness. Note i was so worried that this would do more bad things that I dug around.. and the obvious thing that could go wrong didnt - embryos that are meant to die still do!
